tests/runtime/ray/test_dag.py

```diff
@@ -121,11 +125,9 @@ class TestRayRuntime:
 
     @staticmethod
     @pytest.fixture
-    def runtime_config():
-        return RuntimeConfiguration(
-            config_name="TestRayRuntime",
-            runtime_name=RuntimeName.RAY_LOCAL,
-        )
+    def runtime_config(monkeypatch):
+        monkeypatch.setattr(_dag.RayRuntime, "startup", lambda *_, **__: ...)
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏It might be a bit unexpected for a fixture that returns a config to also mock the startup method.

Reply:
good idea, moved it to separate fixture